packages/slo-stat-bot/cron.yaml

```diff
@@ -0,0 +1,6 @@
+cron:
+  - name: slo_stat_bot
+    schedule: */5 * * * *
+    params:
+      installation:
+        id: 11112564
```

Comment:
FYI, I don't see the installation id in the current payload.

Reply:
It's on the underscored one, not the dasherized (duplicate) job.